Initial release v2.1.5

tescande opened the pull request:
Sources imported from [1]. Spec file created from the one provided in the source archive.

[1] https://sourceforge.net/projects/traceroute/files/

tescande committed:
Initial release v2.1.5

Imported from upstream version 2.1.5 and clean up spec file.

Signed-off-by: Thierry Escande <thierry.escande@vates.tech>

stormi commented on SPECS/traceroute.spec:
```diff
@@ -0,0 +1,52 @@
+%{_mandir}/*/*
+
+%changelog
+* Thu Apr 25 2024 Thierry Escande <thierry.escande@vates.tech> - 2.1.5-1
+- Initial release
```
If this is forked from an upstream SRPM, we should retain their changelog.

stormi replied:
We should also import it, unmodified, in a branch named `upstream`

stormi replied:
I re-read your commit message, which answers this comments before I wrote them :)

tescande committed:
Import traceroute-2.1.5.tar.gz

Sources imported from [1]. Spec file created from the one provided in
the source archive.

[1] https://sourceforge.net/projects/traceroute/files/

Signed-off-by: Thierry Escande <thierry.escande@vates.tech>

stormi commented:
I suggest we test it a bit before merging.

ydirson commented on SPECS/traceroute.spec:
```diff
@@ -0,0 +1,48 @@
+%install
+install -d $RPM_BUILD_ROOT/bin
+install -m755 traceroute/traceroute $RPM_BUILD_ROOT/bin
+pushd $RPM_BUILD_ROOT/bin
+ln -s traceroute traceroute6
+popd
```
we could improve to create a relative symlink without even changing dir with
```sh
ln -sr $RPM_BUILD_ROOT/bin/traceroute $RPM_BUILD_ROOT/bin/traceroute6
```

tescande replied:
Done

ydirson reviewed (approved):
small possible improvement over upstream rules, otherwise lgtm

stormi commented:
Koji build to v8.3-incoming: http://koji.xcp-ng.org/taskinfo?taskID=52173

stormi commented:
New build attempt after adding `gcc` to the `BuildRequires`: http://koji.xcp-ng.org/taskinfo?taskID=52176
